sdk-base/src/main/java/com/mapbox/maps/plugin/annotation/Annotation.kt

```diff
@@ -71,6 +71,11 @@ abstract class Annotation<T : Geometry>(
    */
   var isSelected: Boolean = false
 
+  /**
+   * Unique feature identifier.
+   */
+  val featureIdentifier: String = "annotation_feature_${(FEATURE_IDENTIFIER++)}"
```

Comment:
Can we reuse `id`?

Reply:
hmm, nice, that way we technically don't even need new API if I understand everything correctly